lib/dor_indexing/indexers/descriptive_metadata_indexer.rb

```diff
@@ -85,7 +92,19 @@ def orcids
         DorIndexing::Builders::OrcidBuilder.build(Array(cocina.description.contributor))
       end
 
-      def title
+      def main_title
+        Cocina::Models::Builders::TitleBuilder.main_title(cocina.description.title)
+      end
+
+      def full_title
+        Cocina::Models::Builders::TitleBuilder.full_title(cocina.description.title)
+      end
+
+      def additional_titles
+        Cocina::Models::Builders::TitleBuilder.additional_titles(cocina.description.title)
+      end
+
```

Comment:
as is obvious, these are calling the new cocina-models methods I recently added.